mmdeploy/codebase/mmpose/deploy/pose_detection.py

```diff
@@ -311,15 +310,17 @@ def get_preprocess(self, *args, **kwargs) -> Dict:
     def get_postprocess(self, *args, **kwargs) -> Dict:
         """Get the postprocess information for SDK."""
         component = 'UNKNOWN'
+        params = copy.deepcopy(self.model_cfg.model.test_cfg)
```

Comment:
I think a more general way is to read the content of model_cfg.model.head.decoder, or if we modify the content of mspn codec like modify the `post_process` to `default`, the converted pipeline.json still have megvii instead.

Reply:
post_process would be updated by line 314-321